rundeck-data-model/src/main/java/org/rundeck/app/data/providers/v1/execution/ReferencedExecutionDataProvider.java

```diff
@@ -0,0 +1,18 @@
+package org.rundeck.app.data.providers.v1.execution;
+
+import org.rundeck.app.data.model.v1.execution.RdReferencedExecution;
+import org.rundeck.app.data.model.v1.job.JobData;
+import org.rundeck.app.data.providers.v1.DataProvider;
+
+import java.util.List;
+
+public interface ReferencedExecutionDataProvider extends DataProvider {
+    Long updateOrCreateReference(Long refId, Long seId, Long execId, String status);
+    RdReferencedExecution findByScheduledExecutionId(Long seId);
+    List<JobData> parentList(Long seId, int max);
```

Comment:
This call should retrieve a list of job ids. Fully hydrated job data objects should not be required.

Reply:
Now it retrieves a list of ids